### ISSUE TITLE
Unable to copy legend text to clipboard

### ISSUE THREAD

**Issue by l2dy** [allegro/turnilo#702]
I can not select text from the legend, which makes it impossible to copy text from the legend without invoking DevTools.

**Comment by adrianmroz-allegro** [comment]
You mean line/bar-chart legend? I can do this:
![image](https://user-images.githubusercontent.com/78143552/106444300-db6d6000-647d-11eb-9d4d-4a29eaafdd82.png)

**Comment by l2dy** [comment]
Yes. I tried it on Firefox and Chromium, but could not select those.

**Comment by l2dy** [comment]
Links to reproduce:
- http://localhost:9090/#wiki/4/N4IgbglgzgrghgGwgLzgFwgewHYgFwhLYCmAtAMYAWcATmiADTjTxKoY4DKxaG2A5lHyh+NTDAAO3GhGJC8AM0RRiAXyYYAtsWQ5i+EAFE05APQBVACoBhRiAUQEaYjXkBtUGgCeE/QS36TDTECgYBdgEACi5YACbuoLEwNOhYuASRAIwAInZQzhL4pJkaPn6E6HL0qgC6tUxQEkhoCSDevgb5MgJ2sRDa2FBpBlRw2CQIeZh0wiDBoQRwsbHEsRFlnTFyvRDB5BzpICtQ5MTYfT1MvjKYawQg6oT9EPR4mQAM6p4b/v2BR39BsNfto7KIxjAELQXl4DJFLJkABJTGZ4UDzMJ/dYdAh9AZDHA7PYHAxwE5nC78B5MJCaF74bCQhD1EAqGTbPAeOYhFxnU6k5arOwKaaadCzdrlFZKSH0JhgRAwcrUto/EDaMnJfQsiQQcarbKAgmDfBuECjfWTJjYODaRpwflMaAAJUwACNMPQalc9SRYpxpq8QEsVmtVEA=
- http://localhost:9090/#wiki/4/N4IgbglgzgrghgGwgLzgFwgewHYgFwgBGcATgLQDGAFqWiADTjTxKoY4DKApmhtgOZR82GAgSMMAWy7IcXfCACiaCgHoAqgBUAwgxAAzCAjRcSQvAG1QaAJ4AHeQSnzGJLvoXO9zgAqmsACbmViABMCToWLgEPgCMACJ6UCZ2+GSxEvaOIAjoXMkgAL4AuiWMUHZIaMHWWQrJJBACegEQ0thQUQrUcNjYXAhJmCR0eKBuHgRwAQFcAd51BFD++S0QbhTs0aH5FFzYrc2MDo2Y8wRFjEiSEKOxAAyF9LUOnm0uoe8dXU7vevwRES5Rq2BQ+TSxAASQxG+HG7je0gWrwIrXanRwaw2WwUcCgewOTX4lxybVuwlECDKIGWjVWlnh+lM+z2uJmcz0+mGknQcJAthRO30cFEdEYYEQMGyJIF2WkePC8mpdia/QC8S+GI6+AsIB6fQGemwcGkFTgrMY0AASphCJg6MVjqq5hxhqMQNNZvNCkA=
